apps/condo/domains/notification/schema/Message.test.js

```diff
@@ -45,6 +45,20 @@ describe('Message', () => {
         expect(obj.readAt).toEqual(null)
     })
 
+    test('admin can\'t update organization', async () => {
```

Comment:
Could you please write test for `disconnect` case

Reply:
👍